schemas/cloudevents-v1.0.2.yaml

```diff
@@ -0,0 +1,117 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+description: CloudEvents Specification JSON Schema
+type: object
+properties:
+  id:
+    description: Identifies the event.
+    "$ref": "#/definitions/iddef"
+    examples:
+    - A234-1234-1234
+  source:
+    description: Identifies the context in which an event happened.
+    "$ref": "#/definitions/sourcedef"
+    examples:
+    - https://github.com/cloudevents
+    - mailto:cncf-wg-serverless@lists.cncf.io
+    - urn:uuid:6e8bc430-9c3a-11d9-9669-0800200c9a66
+    - cloudevents/spec/pull/123
+    - "/sensors/tn-1234567/alerts"
+    - 1-555-123-4567
+  specversion:
+    description: The version of the CloudEvents specification which the event uses.
+    "$ref": "#/definitions/specversiondef"
+    examples:
```

Comment:
example or enum ?
If cloud event move to 1.1 or 2.0 we will have to update the spec, I guess. So, it is more than an example.